.pre-commit-config.yaml

```
@@ -15,3 +15,8 @@ repos:
         stages: [commit-msg]
     repo: https://github.com/jorisroovers/gitlint
     rev: v0.18.0
+  - hooks:
+      - id: go-mod-tidy
+    args: [--verbose]
```

Is it providing verbose output? I think the indentation is wrong.
```suggestion
  - hooks:
      - id: go-mod-tidy
        args: [--verbose]
```